package.json

```diff
@@ -20,10 +23,12 @@
     "expo-linking": "^6.3.1",
     "expo-router": "^3.5.16",
     "expo-status-bar": "~1.12.1",
+    "postgres": "^3.4.4",
     "react": "18.2.0",
     "react-native": "0.74.2",
     "react-native-safe-area-context": "^4.10.5",
-    "react-native-screens": "^3.32.0"
+    "react-native-screens": "^3.32.0",
+    "tsx": "^4.12.0"
   },
   "devDependencies": {
```

Comment:
ah seems `devDependencies` is missing SafeQL

you can also set up a GitHub Actions workflow so that we're sure that ESLint and TypeScript aren't failing somehow

Reply:
Added `@ts-safeql/eslint-plugin` and `libpg-query` to the `devDependencies`. [8fe7524](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/8fe75242b1094bcbb26161a38eed5094e6136899)

GH actions workflow also fixed. Last commit [86d565a](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/86d565ae5b0fa4d543c005397e5eb50fc1175f33)

Reply:
looks like the ESLint plugin hasn't been set up completely (Prettier deps missing):

https://github.com/upleveled/eslint-config-upleveled/blob/6c24362091950278f1b94bbc5aaf4de7f5aebb04/bin/install.js#L99-L108